subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/TestTaskJdkRelocationIntegrationTest.groovy

```diff
@@ -27,7 +28,13 @@ import org.gradle.util.internal.TextUtil
 import static org.gradle.util.internal.TextUtil.normaliseLineSeparators
 
 @Requires(IntegTestPreconditions.MoreThanOneJava8HomeAvailable)
-class TestTaskJdkRelocationIntegrationTest extends AbstractTaskRelocationIntegrationTest {
+class TestTaskJdkRelocationIntegrationTest extends AbstractTaskRelocationIntegrationTest implements JavaToolchainFixture {
+
+    def setup() {
+        executer.beforeExecute {
+            withInstallations(Jvm.current(), AvailableJavaHomes.jdk11)
```

Comment:
❓ I'm not sure it needs the current JVM too?

Reply:
Yeah, I think you're right.